pages/monsters/[id].vue

```diff
@@ -4,19 +4,30 @@
     <div class="flex items-end justify-between gap-8">
       <h1 class="flex-auto">{{ monster.name }}</h1>
 
-      <div class="flex flex-none items-start">
-        <button
-          class="flex items-center gap-1 rounded-md p-1 text-xs text-blood outline outline-1 outline-blood hover:bg-blood hover:text-white"
-          @click="toggleMode()"
-        >
+      <div class="flex flex-none items-start gap-2">
+        <!-- <button           class="flex items-center gap-1 rounded-md p-1 text-xs text-blood outline outline-1 outline-blood hover:bg-blood hover:text-white"
```

Comment:
Currently Compact Mode does not work in stating/prod, so I've temporarily hidden the button here